src/20_Components/amp-video.html

```diff
@@ -1,7 +1,7 @@
 <!--
   ## Introduction
 
-  Embed videos into your AMP HTML files. Video source files should be served via HTTPS.
+  Use [`amp-video`](https://www.ampproject.org/docs/reference/components/amp-video) Embed videos into your AMP HTML files. Video source files should be served via HTTPS.
```

Comment:
this 2 sentences don't make sense together. I think you are missing "to embed" instead of "Embed"

Reply:
done